.gitignore

```diff
@@ -90,3 +90,6 @@ venv/
 
 # Rope project settings
 .ropeproject
+
+# Auto/Temp Generated Files by Tools
+*.swp
```

Comment:
I don't think we should include .swp here?

Reply:
I can take it out .. but any reason ?